webapp/client/src/components/HelpModal.js

```diff
@@ -32,9 +32,9 @@ const Help = styled.button`
   }
 `;
 
-const Title = ({ children }) => (
-  <h5 className="modal-title mb-n2">{children}</h5>
-);
+function Title({ children }) {
+  return <h5 className="modal-title mb-n2">{children}</h5>;
+}
```

Comment:
I had to change this because of the new linting rule. Do you see any problem with that?

Reply:
:+1: